contracts/strategies/allocation/linear-qf/qf-offchain-spec.md

```diff
@@ -0,0 +1,60 @@
+Spec: Quadratic Funding with Offchain Calculation
+---------------------------------
+
+## Overview 
+Quadratic Funding (QF) is a category of mechanisms that distribute a funding pool based upon individual donations to eligible recipients. Essentially, the amount that an individual donates to a project is considered their "vote", which is weighted according to the type of QF calculation formula being used. 
+
+At time of writing (June 2023), most prominent QF formulas are too computationally expensive to be calculated on-chain at a reasonable cost. As a step towards an eventual fully on-chain QF solution, this strategy functions a hybrid solution: 
+* votes are recorded **on-chain**
+* the distribution is calculated **off-chain**
+* the final distribution is recorded **on-chain**
+
+In order to facilitate this, the contract will emit vote events so that off-chain calculation services can pull in the relevant data. 
+
+## User flows
+Full flow is documented here: https://miro.com/app/board/uXjVMXyfa-o=/?share_link_id=797514162800
```

Comment:
Can you export this as a png and drop it here in the doc?

Reply:
Good call, added